naga/src/back/wgsl/writer.rs

```diff
@@ -481,7 +481,10 @@ impl<W: Write> Writer<W> {
                         "storage_",
                         "",
                         storage_format_str(format),
-                        if access.contains(crate::StorageAccess::LOAD | crate::StorageAccess::STORE)
+                        if access.contains(crate::StorageAccess::ATOMIC) {
+                            ",atomic"
```

Comment:
Question: Atomic in syntax implies read-write?

Reply:
yes